.gitpod.yml

```diff
@@ -30,6 +30,14 @@ tasks:
       # Open Moodle site in browser.
       gp ports await 8000 && gp preview $(gp url 8000)
 
+      # Change permissions to allow installation of plugins.
+      bin/moodle-docker-compose exec webserver bash -c 'chmod -R 777 .'
+
+      # Add some data if the file exists.
+      if [ -e "moodle/admin/tool/generator/tests/fixtures/gitpod-basic-scenario.feature" ]; then
+        bin/moodle-docker-compose exec webserver php admin/tool/generator/cli/runtestscenario.php --feature=./../tests/fixtures/gitpod-basic-scenario.feature
```

Comment:
Does the `runtestscenario.php` CLI script find feature files relative to it? I imagine it does, but better ask to confirm.

Reply:
Yes, it does :-)